core/src/test/java/org/testcontainers/images/builder/DockerignoreTest.java

```diff
@@ -0,0 +1,56 @@
+package org.testcontainers.images.builder;
+
+import static org.rnorth.visibleassertions.VisibleAssertions.assertTrue;
+import static org.rnorth.visibleassertions.VisibleAssertions.fail;
+
+import java.io.File;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.junit.Test;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.startupcheck.OneShotStartupCheckStrategy;
+
+import com.github.dockerjava.api.exception.DockerClientException;
+
+public class DockerignoreTest {
+    
+    private static final Path INVALID_DOCKERIGNORE_PATH = Paths.get("src/test/resources/dockerfile-build-invalid");
+    
+    @Test
+    public void testInvalidDockerignore() throws Exception {
+        try {
+            new ImageFromDockerfile()
+                .withFileFromPath(".", INVALID_DOCKERIGNORE_PATH)
+                .withDockerfile(INVALID_DOCKERIGNORE_PATH.resolve("Dockerfile"))
+                .get();
+            fail("Should not be able to build an image with an invalid .dockerignore file");
+        } 
+        catch(DockerClientException e) {
+            if (!e.getMessage().contains("Invalid pattern"))
+                throw e;
+        }
+    }
+    
+    @SuppressWarnings("resource")
+    @Test
+    public void testValidDockerignore() throws Exception {
+        ImageFromDockerfile img = new ImageFromDockerfile()
+                .withFileFromPath(".", DockerfileBuildTest.RESOURCE_PATH)
+                .withDockerfile(DockerfileBuildTest.RESOURCE_PATH.resolve("Dockerfile-currentdir"));
+        try (final GenericContainer<?> container = new GenericContainer<>(img.get())
+            .withStartupCheckStrategy(new OneShotStartupCheckStrategy())
+            .withCommand("ls", "/")) 
```

Comment:
If would be nice to format it like here:
https://github.com/testcontainers/testcontainers-java/blob/a74b77f7dcd759f798d36f6b3b124f375c363bcf/core/src/test/java/org/testcontainers/junit/CopyFileToContainerTest.java#L15-L19

Reply:
(maybe this a space between `try` and `(`